application/src/main/java/org/opentripplanner/model/plan/paging/cursor/PageCursorInput.java

```diff
@@ -1,29 +1,44 @@
 package org.opentripplanner.model.plan.paging.cursor;
 
 import java.time.Instant;
+import org.opentripplanner.framework.model.Cost;
 import org.opentripplanner.model.plan.ItinerarySortKey;
 
 /**
- * This class holds information needed to create the next/previous page cursors when there were
- * itineraries removed due to cropping the list of itineraries using the numItineraries parameter.
+ * This class holds information needed to create the next/previous page cursors either when there were
+ * itineraries removed due to cropping the list of itineraries using the numItineraries parameter or
+ * when the {@link org.opentripplanner.routing.algorithm.filterchain.filters.transit.RemoveTransitIfStreetOnlyIsBetter}
+ * filter is used.
  * <p>
  * The Instant fields come from the sets of itineraries that were removed and the ones that were
  * kept as a result of using the numItineraries parameter.
  */
 public interface PageCursorInput {
   /**
-   * The earliest-removed-departure defines the start of the search-window following the
+   * The earliest removed departure defines the start of the search window following the
    * current window. To include this removed itinerary (and all other removed itineraries)
-   * in the next-page search the search windows must overlap.
+   * in the next page search the search windows must overlap.
    */
   Instant earliestRemovedDeparture();
+
+  /**
+   * The latest removed departure together with a search window duration is used to
+   * calculate the start of the search window preceding the current window.
+   */
   Instant latestRemovedDeparture();
 
   /**
-   * In case the result has too many results: The {@code numberOfItineraries} request parameter
-   * is less than the number of itineraries found, then we keep the last itinerary kept and
-   * returned as part of the result. The sort vector will be included in the page-cursor and
-   * used in the next/previous page to filter away duplicates.
+   * If the search has too many results, the {@code numberOfItineraries} request parameter
+   * is less than the number of itineraries found, then we store information from the last
+   * itinerary kept and returned as part of the result. The sort vector will be included
```

Comment:
What is the sort vector? Have used that word anywhere else?

Reply:
FYI: The "sort-vector" is in use in SortOrderComparator and CompositeComparator.